django_scrubber/scrubbers.py

```diff
@@ -130,9 +130,9 @@ def _initialize_data(self):
                     provider_name)
             faker_instance.add_provider(provider)
 
-        logger.info('Initializing fake scrub data for provider %s' % self.provider)
+        logger.info('Initializing fake scrub data for provider %s' % self.provider_key)
```

Comment:
The `provider_key` here is not very helpful, since it has a hash of arguments. Mabye just use the `self.provider` and the args' textual representation?

Reply:
Not sure if I did this right. Just did this on the fly without having run the testsuite on my machine yet. If it fails I'll try more thoroughly tomorrow.